lib/Service/ObjectService.php

```diff
@@ -28,6 +32,100 @@ public function __construct(ObjectEntityMapper $objectEntityMapper, RegisterMapp
 		$this->schemaMapper = $schemaMapper;
 	}
 
+	public function find(int|string $id) {
+		return $this->getObject(
+			register: $this->registerMapper->find($this->getRegister()),
+			schema: $this->schemaMapper->find($this->getSchema()),
+			uuid: $id
+		);
+	}
+
+	public function createFromArray(array $object) {
+		return $this->saveObject(
+			register: $this->getRegister(),
+			schema: $this->getSchema(),
+			object: $object
+		);
+	}
+
+	public function updateFromArray(string $id, array $object, bool $updatedObject) {
+		$object['id'] = $id;
+		return $this->saveObject(
+			register: $this->getRegister(),
+			schema: $this->getSchema(),
+			object: $object
+		);
+	}
+
+	public function delete(array|\JsonSerializable $object): bool
+	{
+		if($object instanceof \JsonSerializable === true) {
+			$object = $object->jsonSerialize();
+		}
+
+		return $this->deleteObject(
+			register: $this->registerMapper->find($this->getRegister()),
+			schema: $this->schemaMapper->find($this->getSchema()),
+			uuid: $object['id']
+		);
+	}
+
+	public function findAll(?int $limit = null, ?int $offset = null, array $filters = []): array
+	{
+		$objects = $this->getObjects(
+			register: $this->getRegister(),
+			schema: $this->getSchema(),
+			limit: $limit,
+			offset: $offset,
+			filters: $filters
+		);
+//		$data = array_map([$this, 'getDataFromObject'], $objects);
+
+		return $objects;
+	}
+
+	public function findMultiple(array $ids): array
```

Comment:
Please make sure this function matches [OpenCatalogi ObjectService getMultipleObjects()](https://github.com/ConductionNL/opencatalogi/blob/d8cd5b365d7bc53493a0ceb9739a6c95c385f7c6/lib/Service/ObjectService.php#L197) exactly or the other way around